frontend/src/components/EditMarkModal/EditMarkModal.tsx

```diff
@@ -1,10 +1,10 @@
 import type { Dispatch, SetStateAction } from 'react';
 import React, { useState } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
-import {
-  Button, Input, message,
-  Modal,
-} from 'antd';
+import Button from 'antd/lib/button';
```

Comment:
I know importing the component directly is our only option to minimise build sizes but it still weird to import it like this rippp

Reply:
Actually - I'm not sure if this is the case since tree shaking _should_ be a thing. I'll have to test it with the previous way of importing and check out if the tree shaking is actually in effect.